app/src/main/java/com/jxc/app/models/Article.java

```diff
@@ -0,0 +1,21 @@
+package com.jxc.app.models;
+
+import java.util.List;
+
+import lombok.*;
```

Comment:
On évite les imports *, ça pèse lourd, on préfère importer uniquement ce dont on a besoin. Automatiquement, IntelliJ te converti 5 imports ou plus d'une même librairie en import *, à changer dans les options d'IntelliJ : 
Settings > Editor > Code Style > Java > Imports > Class count to use import with '*'